lib/rollup.config.js

```diff
@@ -6,6 +6,30 @@ import pkg from "./package.json";
 import commonPkg from "../package.json";
 
 const components = require("./src/index.json");
+
+function injectCSS(css, uniqueId, isPrepend) {
+    if (!css || !document) {
```

Comment:
Should use `typeof document === 'undefined'` as per [this PR](https://github.com/sebgroup/react-components/pull/756).

Reply:
https://github.com/sebgroup/react-components/pull/761#discussion_r943061367 :)